neuralpredictors/data/transforms.py

```diff
@@ -386,6 +389,8 @@ def __call__(self, x):
             "responses": self.transforms["responses"](key_vals["responses"]),
```

Comment:
why apply transforms given that `transforms` for `responses`, `behavior` and `pupile_center` are set as no-op. Do you expect these values to be overridden?